source/Window.cpp

```diff
@@ -98,7 +85,11 @@ void Window::create(const char *title) {
 
   context = SDL_GL_CreateContext(window);
 
-  initEpoxy();
+  if (config.getIgnoreGlVersion()) {
+    Util::Log(Warning, "Window") << "Ignore OpenGl version";
+  } else {
+    opengl::initGL();
+  }
```

Comment:
the sole purpose of "IgnoreGlVersio" is to force game to run even when the opengl version is too low or not detected